### PR TITLE
Add player cams link page

### PR DESCRIPTION
**Pull request by ZusorCode**

### CODE REVIEW

File: server/src/actions/create-live-guest.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                "Avatar": `https://cdn.discordapp.com/avatars/${user.discord.id}/${user.discord.avatar}.webp?size=512`
+            });
+            return response?.error ? error("Airtable error", 500) : success();
+        } else {
+            let response = await createRecord("Live Guests", {
+                "Discord ID": user.discord.id,
+                "Avatar": `https://cdn.discordapp.com/avatars/${user.discord.id}/${user.discord.avatar}.webp?size=512`,
+                "Name": user.discord.username,
+                "Player": [user.airtable.id]
+            });
```
Comment (slmnio) [inline server/src/actions/create-live-guest.js:28]: Could also set `Use Cam` to `true` since it's likely they're about to use their webcam

Comment (ZusorCode) [reply server/src/actions/create-live-guest.js]: Fixed in 2dcab6f4e62a7ec14d2a7d9c5b1f1da1562c5e55

**Comment by ZusorCode** [comment]
We should also add a cam link to the detailed match page for the casters

File: website/src/views/sub-views/Cams.vue [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            });
+        },
+        isAuthed() {
+            return this.$root.auth.user;
+        }
+    },
+    async mounted() {
+        this.liveGuestInterval = setInterval(async () => {
+            await this.createLiveGuest();
+        }, 1000);
```
Comment (slmnio) [inline website/src/views/sub-views/Cams.vue:78]: don't think you need this anymore

**Review by slmnio — APPROVED** [review]
Changed some of the styling to use Bootstrap buttons and added a line of text. Server looks good, just removed a console.log but otherwise looks fine. We'll need to think whether moving the .webcam to slmn.gg/cams is worth it or not - at the moment it's probably not but it will make it easier to use externally.

**Pull request merged**